pkg/cli/admin/catalog/mirror.go

```diff
@@ -173,8 +181,12 @@ func NewMirrorCatalog(f kcmdutil.Factory, streams genericclioptions.IOStreams) *
 	flags.StringVar(&o.FromFileDir, "from-dir", o.FromFileDir, "The directory on disk that file:// images will be read from. Overrides --dir")
 	flags.IntVar(&o.MaxPathComponents, "max-components", 2, "The maximum number of path components allowed in a destination mapping. Example: `quay.io/org/repo` has two path components.")
 	flags.StringVar(&o.IcspScope, "icsp-scope", o.IcspScope, "Scope of registry mirrors in imagecontentsourcepolicy file. Allowed values: repository, registry. Defaults to: repository")
+	flags.StringVar(&o.IdmsScope, "idms-scope", o.IdmsScope, "Scope of registry mirrors in imagedigestmirrorset file. Allowed values: repository, registry. Defaults to: repository")
 	flags.IntVar(&o.MaxICSPSize, "max-icsp-size", maxICSPSize, "The maximum number of bytes for the generated ICSP yaml(s). Defaults to 250000")
+	flags.IntVar(&o.MaxIDMSSize, "max-idms-size", maxIDMSSize, "The maximum number of bytes for the generated IDMS yaml(s). Defaults to 250000")
 	flags.BoolVar(&o.ContinueOnError, "continue-on-error", true, "If an error occurs while mirroring, keep going and attempt to mirror as much as possible.")
+	flags.MarkDeprecated("icsp-scope", "support for it will be removed in a future release.Use --idms-scope instead.")
```

Comment:
nit: would be better to also change examples https://github.com/openshift/oc/blob/49ed8f974f377394d9ca3887dda06daafc7fcb47/pkg/cli/admin/catalog/mirror.go#L82